pkg/providers/vsphere/session/session_vm_update.go

```diff
@@ -44,6 +46,13 @@ type VMUpdateArgs struct {
 	NetworkResults network2.NetworkInterfaceResults
 }
 
+// VMResizeArgs contains the arguments needed to resize a VM on VC.
+type VMResizeArgs struct {
+	ConfigSpec vimtypes.VirtualMachineConfigSpec
+	VMClass    *vmopv1.VirtualMachineClass
```

Comment:
Nit: Is this needed for later in the args?

Reply:
Yes we'll need it for annotation